modules/olm-installing-opm.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-managing-custom-catalogs.adoc
+
+[id="olm-installing-opm_{context}"]
+= Installing `opm`
+
+To install the Operator Package Manager (`opm`) CLI tool on your workstation:
```

Comment:
Did we ever officially define this initialism? I kind of liked leaving it undefined 😄

Reply:
Dropping the 2 uses of "Operator Package Manager" for now.